src/Components/Artwork/Details.tsx

```diff
@@ -175,19 +174,12 @@ export class Details extends React.Component<Props, null> {
     return (
       <ContextConsumer>
         {({ user }) => {
-          const hasLabFeature =
-            user &&
-            user.lab_features &&
-            user.lab_features.includes("New Buy Now Flow")
-          const enableBuyNowFlow = sd.ENABLE_NEW_BUY_NOW_FLOW || hasLabFeature
-
           return (
             <div>
-              {enableBuyNowFlow && this.saleInfoLine()}
+              {this.saleInfoLine()}
               {this.artistLine()}
               {this.titleLine()}
               {this.partnerLine()}
-              {!enableBuyNowFlow && this.props.showSaleLine && this.saleLine()}
```

Comment:
Why remove `this.saleLine()`?

Reply:
I think this is correct. When the flow is not enabled, the older style brick has the sale/info line at the bottom of the brick (and itself can be optionally hidden in non-commercial contexts via the `showSaleLine` prop).

Since we're rolling out the new brick style everywhere (with sale info at the top), this line just goes away.